sources/pmwebapi_test.go

```diff
@@ -23,6 +23,13 @@ type AutoGenerated struct {
 	URL    string `json:"url"`
 }
 
+type testlables struct {
+	name     string
+	tyype    string
```

Comment:
Why is this `tyype`?

Reply:
when i wrote type it gave me an error, type is specific in golang i think